1.0/docs/release-notes.md

```diff
@@ -15,6 +15,295 @@ subtitle: About this release
   }
 </style>
 
+
+## [Release 1.3.0](https://github.com/Polymer/polymer/tree/v1.3.0) (2016-02-22) {#v1-3-0}
+
+This release includes the following new features:
+
+-   [#2348](https://github.com/Polymer/polymer/issues/2348). Allow late
+    upgrading of bound elements. Fixed an issue where setting properties on an
+    un-upgraded element Polymer's setters, preventing property effects (bindings,
+    observers, and computed properties) from working.
+
+-   [#3023](https://github.com/Polymer/polymer/issues/3023). Add support for
+    `[attribute~="value"]` selector.
+
+-   [#3299](https://github.com/Polymer/polymer/issues/3299). Support dynamic
+    functions for computed annotations. See
+    [Support for dynamic functions in bindings](#1.3.0-dynamic-fns) for details.
+
+-   [#2399](https://github.com/Polymer/polymer/pulls/2399). Add support for
+    custom properties in `@keyframes` rules.
+
+-   [#2486](https://github.com/Polymer/polymer/pulls/2486). Allow newlines in
+    computed binding argument list. For example:
+
+        <div hidden$="[[_someVeryLongFunctionName(
+            someVeryLongArg,
+            anotherCrazyLongArg)]]">
+
+
+This release fixes the following issues:
+
+-   [#1715](https://github.com/Polymer/polymer/issues/1715). Allow two-way binding
+    for negated expressions. (For example, `checked="{%raw%}{{!hidden}}{%endraw%}"`.)
+
+
+-   [#1727](https://github.com/Polymer/polymer/issues/1727). Eliminate the need
+    to add `:host` or other selector in front of `::content` in some situations.
+    This applies if the `<content>` tag is a top-level local DOM child, and there
+    are no other local DOM nodes that could match the selector, for example:
+
+        <template>
+          <content></content>
+        </template>
```

Comment:
Maybe add an example where you would still need to use a wrapper around `<content>`?